### ISSUE TITLE
[A11Y] Language selector markup

### ISSUE THREAD

**Issue by foxbunny** [indico/indico#5899]
**Describe the bug**

The language selector is inaccessible due to missing `href` on the anchor tags. Furthermore it is semantically incorrect as language switching is not linking to anything.

**To Reproduce**
1. Go to main page
2. Attempt to tab into the language selector
3. Observe that tab navigation skips over the language links

**Expected behavior**

The language links/buttons are focusable and operable using keyboard.

**Additional context**

- https://talk.getindico.io/t/keyboard-language-selector/3219
- https://talk.getindico.io/t/language-language-selector/3207
- https://talk.getindico.io/t/instructions-language-selector/3211
- https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html
- https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions
- https://www.w3.org/WAI/WCAG21/Understanding/language-of-page.html

**Comment by foxbunny** [comment]
As part of this fix, I will address several other issues I have identified earlier.
